src/docs/cmdstan-guide/installation.tex

```diff
@@ -644,6 +672,10 @@ \subsection{GNU Make Build Tool}
 \item Tested Versions: 3.81 (Mac OS X), 3.79 (Windows 7)
 \end{itemize}
 %
+On Windows the \code{mingw32-make} variant must be used, since the
```

Comment:
"On Windows \code{mingw32-make} must be used, as it is a requirement to build Intel TBB. This variant of make is available..."